src/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -263,7 +263,11 @@ Interop.NetSecurityNative.Status status
                     tokenBytes = tokenBytes.Slice(0, length - 1);
                 }
 
+#if NETSTANDARD2_0
+                return Encoding.UTF8.GetString(tokenBytes.ToArray(), 0, tokenBytes.Length);
```

Comment:
If we are OK with using unsafe code here, the ptr overload of `GetString` would avoid the extra allocation/cost.

@davidsh - is this code perf sensitive? There is some unsafe code already (e.g. System.Net.Security.Native/Interop.GssBuffer.cs from https://github.com/dotnet/corefx/pull/42522), so I would think use of ptrs here (within the #ifdef) would be fine too (if worth the trade-off).

Reply:
I was too worried about making this method unsafe.  I just wanted to get the build break fixed. :)  So, I'm fine with the .ToArray(). In fact, before my PR which broke this, it already was doing the equivalent anyways. Now, at least on 'netcoreapp' build, it will use Span completely.

This code is not perf sensitive. But I will think about your optimization suggestion and perhaps do a follow-up PR.

Reply:
Ok.